gke/standard/zonal/secondary-boot-disk/main.tf

```diff
@@ -0,0 +1,47 @@
+/**
+* Copyright 2024 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+# [START gke_standard_secondary_boot_disk_zonal]
+resource "google_container_cluster" "default" {
+  name               = "default"
+  location           = "us-central1-a"
+  initial_node_count = 1
+  # Set `deletion_protection` to `true` will ensure that one cannot
+  # accidentally delete this instance by use of Terraform.
+  # Set `min_master_version` because secondary_boot_disks require GKE 1.28.3-gke.106700 or later.
```

Comment:
nit: let's move this comment and the `min_master_version` parameter after `initial_node_count`.  That way the comments are above their respective parameters, and wee try to keep `deletion_protection = false` as the final parameter in the block.

Reply:
Moved deletion_protection to bottom